codemods/transforms/use-resource-imports/index.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  getImportDeclaration,
+  removeImportDeclaration,
+} from '@codeshift/utils';
+import type { FileInfo, API } from 'jscodeshift';
+
+const resourcesImportSpecifiers = [
+  'createResourcesPlugin',
+  'ResourceSubscriber',
+  'useResource',
+  'addResourcesListener',
+  'createResource',
+  'useResourceStoreContext',
+  'ResourceDependencyError',
+  'getResourceStore',
+  'ResourceStore',
+  'CreateResourceArgBase',
+  'CreateResourceArgSync',
+  'CreateResourceArgAsync',
+  'RouteResources',
+  'ResourceStoreContext',
+  'ResourceStoreData',
+  'RouteResource',
+  'RouteResourceError',
+  'RouteResourceLoading',
+  'RouteResourceResponse',
+  'RouteResourceUpdater',
+  'RouterDataContext',
+  'UseResourceHookResponse',
+  'mockRouteResourceResponse',
+];
+
+const packageName = 'react-resource-router';
+const resourcesPackageName = 'react-resource-router/resources';
+
+export default function transformer(
+  file: FileInfo,
+  { jscodeshift: j }: API
+): string | undefined {
+  const source = j(file.source);
+
+  const rrrImportDeclaration = getImportDeclaration(j, source, packageName);
```

Comment:
This won't work if the path is defined multiple times; not an issue if we rely on ESLint rules such as [no-duplicates](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/)